tests/resources/tmp_download_test/remote.wav

```diff
@@ -0,0 +1 @@
+This is a test file. It is validated using checksum. Do not change the contents of this file.
```

Comment:
please remove this test file. I do not think it should be here at all, right @harshpalan ?

Reply:
I think this file is only meant to be used in local tests, but it is not needed for github builds/tests. I'll remove it.